### DISCUSSION TITLE
Rust Custom Map ?

### DISCUSSION THREAD

**Discussion by HerrDummkopf** [GameServerManagers/LinuxGSM#3750] [Q&A]
Hi ho :)

my question is, is there a possibility to install a custom map? 

I've already tried that, but it doesn't work. 

1. Upload your .map file to Dropbox.
2. Once uploaded, click the paperclip icon next to the map file. This will copy the download link.
3. select Startup Parameters
4. In the Additional Argument field, add "+levelurl [your download link]"
       Important: At the end of the link, make sure you replace the "dl=0" with "dl=1".
        Example: map link

I tried both start parameters     +server.levelurl \"${customlevelurl}\    |    +levelurl [your download link]

After one minute the server is automatically off
________________________________________________

Thank you in advance for the help

**Comment by johnoclockdk** [comment]
have you change the url to have the dl. in dropbox.com

dl.dropbox.com

**Comment by HerrDummkopf** [comment]
So the address was     https://www.dropbox.com/sh/*******/************************?dl=1
